### PR TITLE
SW-4498 Add search API call to get projects for given species, add to redux

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #1995** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1995?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **PR #1996** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1996?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/1995?utm_source=stack-comment).

File: src/services/SpeciesService.ts [hunk 1]
```diff
@@ -336,24 +342,56 @@ const suggestSpecies = async (organizationId: number, query: string): Promise<Su
   return (await SearchService.search(params)) as SuggestedSpecies[] | null;
 };
 
+const getSpeciesProjects = async (
```
Comment (karthikbtf) [inline src/services/SpeciesService.ts:345]: remove async or add an await to the search call?

File: src/services/SpeciesService.ts [hunk 2]
```diff
@@ -336,24 +342,56 @@ const suggestSpecies = async (organizationId: number, query: string): Promise<Su
...
+  getSpecies,
   getSpeciesDetails,
   getSpeciesNames,
+  getSpeciesProjects,
+  getSpeciesUploadStatus,
+  ignoreProblemSuggestion,
+  resolveSpeciesUpload,
   suggestSpecies,
+  updateSpecies,
+  uploadSpecies,
```
Comment (karthikbtf) [inline src/services/SpeciesService.ts:394]: thanks :)

**Review by karthikbtf — APPROVED** [review]
Clear to me. One comment.

**Comment by nickgraz** [comment]
### Merge activity

* **Jan 5, 3:26 PM**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/1995).

**Pull request merged**